src/PaymentMethods/Mbway.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mollie\WooCommerce\PaymentMethods;
+
+class Mbway extends AbstractPaymentMethod implements PaymentMethodI
+{
+    public function getConfig(): array
+    {
+        return [
+            'id' => 'mbway',
+            'defaultTitle' => 'MB Way',
+            'settingsDescription' => '',
+            'defaultDescription' => '',
+            'paymentFields' => false,
+            'instructions' => false,
```

Comment:
We were discussing with Niklas that maybe is better to have consistency in the fields.. maybe we can put here also SEPA => false